galaxy/templates/deployment-job.yaml

```diff
@@ -58,9 +66,26 @@ spec:
 {{- end }}
             - name: GALAXY_CONFIG_DATABASE_CONNECTION
               value: postgresql://{{ .Values.postgresql.galaxyDatabaseUser }}:$(GALAXY_DB_USER_PASSWORD)@{{ template "galaxy-postgresql.fullname" . }}/galaxy
-          command: ["python", "/galaxy/server/scripts/galaxy-main"]
-          args: ["-c", "/galaxy/server/config/galaxy.yml", "--server-name", "$(POD_NAME)", "--attach-to-pool", "job-handlers"]
+          command: ['sh', '-c', '
+          {{- range $key, $entry := .Values.initScripts -}}
+          {{- if (not (eq $key "mountPath")) -}}
+          {{- if $entry.applyToJob -}}
+          sh {{ $.Values.initScripts.mountPath }}/{{ $key }}{{ ".sh && " }}
```

Comment:
since we are already passing this through `sh -c`, the second `sh` seems unnecessary?

Reply:
Correct me if this doesn't make sense, but I think if we don't add the `sh` command to run each script it will execute it based on the first line (which would be desirable to allow other types of scripts if we want that), but will also require us changing the permissions on the file to make it executable: `sh: 1: /galaxy/server/scripts/initscripts/myFirstScript.sh: Permission denied`
It shouldn't be hard to do if that's what we want, so i guess the question is do we want that?